src/guidescanpy/flask/templates/grna_design.html

```diff
@@ -198,6 +216,27 @@ <h5 class="card-header">gRNA Design Tool</h5>
                                         </label>
                                     </div>
                                 </div>
+                                <div class="form-group col-md-6">
+                                    <div class="form-check">
+                                        <input class="form-check-input" type="checkbox" id="checkFilterGRNAPattern"
+                                               name="checkFilterGRNAPattern">
+                                        <label class="form-check-label" for="checkFilterGRNAPattern">
+                                            gRNA pattern to avoid (max 5 nucleotides)
+                                            <i class="fa-regular fa-circle-question" data-toggle="tooltip" data-placement="top"
+                                                title="Avoid gRNA sequences that contain the input pattern.
+&#8226; You can have Ns or Vs in the pattern in addition to A/C/T/G.
+&#8226; All Ns are expanded to A/C/T/G, with all occurrences of N matching the same nucleotide.
+&#8226; All Vs are expanded to A/C/G, with all occurrences of V matching the same nucleotide.
+&#8226; The entire pattern cannot be more than 5 characters long. (This is mostly to avoid exploding the computation, since one simple pattern on the part of the user would end up in a complex pattern on our end).">
```

Comment:
That comment was meant for you :). I think we can just say "Max 5 characters" at the beginning of the help string.

Reply:
Since we already have "(max 5 nucleotides)" in the label we can get rid of this line I think.

Reply:
Sure. It's clearly enough to have "(max 5 nucleotides)". Thanks!